### PR TITLE
Models hub m11.1 (fixes)

### PR DESCRIPTION
**Pull request by gadde5300**
Fixing typo in `bert_sequence_classifier_sms_spam` ,
Fixing python code in `bert_token_classifier_ner_cellular`

#### Updated following redl models:
- docs/_posts/HashamUlHaq/2021-02-04-redl_bodypart_direction_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_bodypart_problem_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_bodypart_procedure_test_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_chemprot_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_clinical_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_date_clinical_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_drug_drug_interaction_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_human_phenotype_gene_biobert_en.md
- docs/_posts/HashamUlHaq/2021-02-04-redl_temporal_events_biobert_en.md
- docs/_posts/HashamUlHaq/2021-07-12-redl_ade_biobert_en.md
- docs/_posts/albertoandreottiATgmail/2021-09-10-redl_bodypart_procedure_test_biobert_en.md
- docs/_posts/muhammetsnts/2021-06-01-redl_bodypart_direction_biobert_en.md
- docs/_posts/muhammetsnts/2021-06-01-redl_bodypart_problem_biobert_en.md
- docs/_posts/muhammetsnts/2021-06-01-redl_bodypart_procedure_test_biobert_en.md
- docs/_posts/muhammetsnts/2021-07-24-redl_chemprot_biobert_en.md
- docs/_posts/muhammetsnts/2021-07-24-redl_clinical_biobert_en.md
- docs/_posts/muhammetsnts/2021-07-24-redl_drug_drug_interaction_biobert_en.md
- docs/_posts/muhammetsnts/2021-07-24-redl_human_phenotype_gene_biobert_en.md
- docs/_posts/muhammetsnts/2021-07-24-redl_temporal_events_biobert_en.md

## Motivation and Context

Fixing the issues ([722](https://github.com/JohnSnowLabs/streamlit-demo-apps/issues/722), [743](https://github.com/JohnSnowLabs/streamlit-demo-apps/issues/743#event-5602474914))
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by HashamUlHaq** [comment]
Hi @gadde5300 you have changed all ner models to ner_jsl_enriched. which is not correct. 

- For the chemprot model, keep the ner model to "ner_chemprot_clinical" and do not define relation pairs.
- For redl_clinical, the ner model should be "ner_clinical" and set these relations: ["problem-test", "problem-treatment"]
- in redl_date correct "delative" to "relative" on line 78
- ner model in redl_drug_drug interaction is also not correct.
- ner model for redl_temporal_events is also not correct.

Please refer to these notebooks and change all ner models and relation pairs properly:
https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/10.Clinical_Relation_Extraction.ipynb
https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/10.1.Clinical_Relation_Extraction_BodyParts_Models.ipynb

**Review by josejuanmartinez — APPROVED** [review]
Everything looks good to me, @gadde5300 . Thank you so much!

File: docs/_posts/murat-gunay/2021-11-04-classifierdl_use_sentiment_pipeline_tr.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This pretrained pipeline identifies the sentiments (positive or negative) in Tur
 
 {:.btn-box}
 <button class="button button-orange" disabled>Live Demo</button>
-<button class="button button-orange" disabled>Open in Colab</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/1.Clinical_Named_Entity_Recognition_Model.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-04-classifierdl_use_sentiment_pipeline_tr.md:24]: I don't think this is correct. classifier_dl_use_sentiment_pipeline is not an NER model.

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Hi @gadde5300 !

I added one request, since I see a strange colab notebook (ner) for a classifier dl pipeline.

**Review by josejuanmartinez — APPROVED** [review]
Looks good now. Thank you!

**Pull request merged**